src/utils/findpackage/packageunix.rs

```diff
@@ -17,25 +18,41 @@ use super::{
 
 // here is the logic of findpackage on linux
 //
-const PREFIXS: [&str; 2] = ["/usr", "/usr/local"];
+static PREFIXES: LazyLock<Vec<String>> = LazyLock::new(|| {
+    if let Ok(command) = Command::new("cmake").arg("--system-information").output() {
```

Comment:
```suggestion
    let Ok(command) = Command::new("cmake").arg("--system-information").output() else {
       return None
    }
```
like this way. the var should be maybe CMAKE_EXE_PREFIXS, like this kind

and if the value is not none, then in get_prefix function, for example, this one in packageunix.rs, if it is not empty, use the prefix provided by cmake, early return, if it is none or empty, use the origin way. This is what I am thinking

Reply:
and I think this variable should in `src/utils/findpackage.rs`, becaused it can be used in all platforms